http-api/src/main/java/bisq/http_api/web_socket/util/GrizzlySwaggerHttpHandler.java

```diff
@@ -51,16 +51,41 @@ public void service(Request request, Response response) throws Exception {
             }
 
             response.setStatus(200);
-            URL resourceUrl = classLoader.getResource(resourceName);
-            if (resourceUrl != null) {
-                Path path = Paths.get(resourceUrl.toURI());
-                response.setContentLengthLong(Files.size(path));
-            }
-            inputStream.transferTo(response.getOutputStream());
+            
+            // Read all bytes to determine content length instead of using Paths which causes a crash in most cases
+            byte[] data = inputStream.readAllBytes();
```

Comment:
inputStream.readAllBytes() loads the entire resource into memory as a byte[]

Better:

```
byte[] buffer = new byte[8192];
    int bytesRead;
    while ((bytesRead = inputStream.read(buffer)) != -1) {
        outputStream.write(buffer, 0, bytesRead);
    }
    ```

Reply:
Yes, we can leave it.